apk-generator/v2/scripts/build_apk.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
-./gradlew assembleRelease --info
+./gradlew assemble$3Release --info
```

Comment:
@niranjan94 @arp95 I have done some research and tested it. If i run `assembleGoogleplay` then i'm getting both debug and release apks.
![build](https://user-images.githubusercontent.com/18264684/28152548-bb208b7a-67be-11e7-9827-2c2e89c1b35f.png)
Ref : https://developer.android.com/studio/build/build-variants.html#product-flavors